Lib/random.py

```diff
@@ -110,6 +110,7 @@ def seed(self, a=None, version=2):
         """
 
         if version == 1 and isinstance(a, (str, bytes)):
+            a = a.decode('latin-1') if isinstance(a, bytes) else a
```

Comment:
Maybe instead convert a string to a list of integers and remove `ord()` in the following code?
```
if isinstance(a, str):
    a = list(map(ord, a))
```
This would make the following code clearer and maybe slightly faster.

Reply:
I prefer to leave the current code alone and just add a conversion for the *bytes* which is so uncommon that no one ever noticed the lack of bytes support.